examples/determinator-rails/config/initializers/determinator.rb

```diff
@@ -1,10 +1,24 @@
 require 'determinator/retrieve/routemaster'
+require 'determinator/cache/active_support_memory_store'
 
-Determinator.configure(
-  retrieval: Determinator::Retrieve::Routemaster.new(
-    discovery_url: 'https://florence.dev/'
-    retrieval_cache: ActiveSupport::Cache::MemoryStore.new(expires_in: 1.minute)
-  )
-  # The following would allow tracking of errors in NewRelic
-  # errors: -> error { NewRelic::Agent.notice_error(error) }
-)
+retrieval = Determinator::Retrieve::Routemaster.new(discovery_url: 'https://flo.dev/')
+Determinator.configure(retrieval: retrieval)
+Determinator.retrieval_cache = Determinator::Cache::ActiveSupportMemoryStore.new(expires_in: 1.minute)
```

Comment:
Any reason to break current configs here?

Reply:
Yeah, I've started storing `Feature` objects rather than whatever gets returned from Routemaster, so it should be much more flexible :)